cmd/admin-handlers-users.go

```diff
@@ -1767,7 +1767,7 @@ func (a adminAPIHandlers) ListPolicyMappingEntities(w http.ResponseWriter, r *ht
 	writeSuccessResponseJSON(w, econfigData)
 }
 
-// AttachPolicyBuiltin - POST /minio/admin/v3/idp/builtin/attach
+// AttachPolicyBuiltin - POST /minio/admin/v3/idp/builtin/policy/attach
```

Comment:
Is it like API end point was always like this `POST /minio/admin/v3/idp/builtin/policy/attach`? Just correction in comment?

Reply:
Yes here it's just a correction of the comment